### PR TITLE
Add a chi example for sync users

### PR DESCRIPTION
**Pull request by ccsxs**
I submit a yuml template for migrating users when expanding capacity, so that people in need can use it.

### CODE REVIEW

File: docs/chi-examples/02-templates-07-syncUser.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: "sync-users"
+spec:
+  defaults:
+    templates:
+      podTemplate: clickhouse
+      dataVolumeClaimTemplate: localpvc-storage
```
Comment (Slach) [inline docs/chi-examples/02-templates-07-syncUser.yaml:9]: remove unnecessary details, 
let's rename to `data-pvc-template`

File: docs/chi-examples/02-templates-07-syncUser.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+           <user_directories replace="replace">
+             <users_xml>
+               <path>users.xml</path>
+             </users_xml>
+             <replicated>
+               <zookeeper_path>/clickhouse/access</zookeeper_path>
+             </replicated>
+           </user_directories>
+        </clickhouse>
+    zookeeper:
```
Comment (Slach) [inline docs/chi-examples/02-templates-07-syncUser.yaml:23]: lets add comment 
```yaml
# deploy zookeeper separatelly, look examples in https://github.com/Altinity/clickhouse-operator/blob/master/deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml
```

File: docs/chi-examples/02-templates-07-syncUser.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            - name: clickhouse
+              image: clickhouse-server:22.10.3.27
+              imagePullPolicy: IfNotPresent
+              volumeMounts:
+                - mountPath: /var/lib/clickhouse
+                  name: local-pvc-my
+    volumeClaimTemplates:
+      - name: local-pvc-my
+        spec:
+          storageClassName: my-localpv
```
Comment (Slach) [inline docs/chi-examples/02-templates-07-syncUser.yaml:47]: remove storageClassName, this is unnecessary

File: docs/chi-examples/02-templates-07-syncUser.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            ckop: my
+          containers:
+            - name: clickhouse
+              image: clickhouse-server:22.10.3.27
+              imagePullPolicy: IfNotPresent
+              volumeMounts:
+                - mountPath: /var/lib/clickhouse
+                  name: local-pvc-my
+    volumeClaimTemplates:
+      - name: local-pvc-my
```
Comment (Slach) [inline docs/chi-examples/02-templates-07-syncUser.yaml:45]: rename to `data-pvc-template`

**Review by Slach — CHANGES_REQUESTED** [review]

**Comment by ccsxs** [comment]
@Slach Hello, I have revised and submitted it according to your comments,thanks.

**Review by Slach — APPROVED** [review]

**Comment by Slach** [comment]
@ccsxs thanks for contribution

**Pull request merged**